src/mlmc/sample_storage.py

```diff
@@ -269,5 +293,39 @@ def unfinished_ids(self):
     def get_level_ids(self):
         return list(self._results.keys())
 
+    def get_chunks_info(self, level_id, i_chunk):
```

Comment:
Seems like too many getters to me. Where we need all these. Should rather be properties where possible.